sqlitecluster/SQLite.cpp

```diff
@@ -282,8 +284,26 @@ void SQLite::_sqliteLogCallback(void* pArg, int iErrCode, const char* zMsg) {
     // This is sort of hacky to parse this from the logging info. If it works we could ask sqlite for a better interface to get this info.
     if (SStartsWith(zMsg, "cannot commit")) {
         // 17 is the length of "conflict at page" and the following space.
-        const char* offset = strstr(zMsg, "conflict at page") + 17;
-        _conflictPage = atol(offset);
+        const char* pageOffset = strstr(zMsg, "conflict at page") + 17;
```

Comment:
Start at `offset` rather than `zMsg`

Reply:
am I missing something here? This was the previous offset property, so we don't have an offset to use on `pageOffset`. 

`tableOffset` below is starting on `pageOffset` already, if that's what you mean

Reply:
Oops, my mistake, I thought I was looking at the next line where we did this.